scripts/release/build-commands/print-post-build-summary.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const chalk = require('chalk');
+const {getUnexecutedCommands} = require('../utils');
+
+const CHANGELOG_PATH =
+  'https://github.com/facebook/react/edit/master/CHANGELOG.md';
+
+module.exports = params => {
+  const command =
+    `./publish.js -v ${params.version}` +
+    (params.path ? ` -p ${params.path}` : '') +
+    (params.dry ? ' --dry' : '');
+
+  console.log(
+    chalk`
```

Comment:
Is this valid syntax? I've never seen it before.

Reply:
Yeah, they're called tagged template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Libs like `styled-components` make heavy use of it, eg:
```js
const Button = styled.button`
	/* Adapt the colours based on primary prop */
	background: ${props => props.primary ? 'palevioletred' : 'white'};
	color: ${props => props.primary ? 'white' : 'palevioletred'};

	font-size: 1em;
	margin: 1em;
	padding: 0.25em 1em;
	border: 2px solid palevioletred;
	border-radius: 3px;
`;
```